db/area_city.sql

```diff
@@ -20,7 +20,7 @@ Date: 2020-02-17 13:21:46
 -- ----------------------------
 DROP TABLE IF EXISTS "public"."area_city";
 CREATE TABLE "public"."area_city" (
-"id" int4 DEFAULT nextval('area_city_id_seq') PRIMARY KEY,

```

Comment:
https://www.postgresql.org/docs/9.1/datatype-numeric.html
https://www.postgresqltutorial.com/postgresql-serial/

No need to have special sequence for PK generation.